### PR TITLE
A couple of new games and an update

### PR DESCRIPTION
**Pull request by AdmiringWorm**
**New Games**
- ALLTYNEX Second
- Alter Ego
- Aura Kingdom
- Car Mechanic Simulator 2014
- Car Mechanic Simulator 2015
- Freight Tycoon Inc.
- HuniePop
- Jolly Rover
- The Raven - Legacy of a Master Thief
- Rime Berta
- StreetHacker
- This War Of Mine
- Without Within
- The Wolf Among Us

**Updated Games**
- Age of Empires - Added missing files to save
- Anno 1404 - Added alternative save location
- Civilization 5 - Added Settings type
- Euro Truck Simulator - Added a new savegames folder
- Game Dev Tycoon - Added alternative save location
- Lost Horizon - Changed to Backup ini files as _Settings_ type

### CODE REVIEW

File: new.xml [hunk 1]
```diff
@@ -188,4 +188,64 @@
...
+        <path ev="steamcommon" path="Alter Ego"/>
+        <registry root="local_machine" key="SOFTWARE\viva media\alter ego us" value="installfolder"/>
+      </locations>
+      <files>
+        <include path="saves" filename="*.egs"/>
+      </files>
+      <files type="Settings">
+        <include path="saves" filename="options" />
+      </files>
+      <linkable />
```
Comment (sanmadjack) [inline new.xml]: This needs changed to 

```
<linkable path="saves" />
```

**Comment by sanmadjack** [comment]
One change needed, see the other comment.

**Comment by AdmiringWorm** [comment]
I've updated the pull request with quite a few new/updated games, if you wouldn't mind too much checking it out?

**Comment by sanmadjack** [comment]
Sorry, I didn't see this request. I'll go over these soon.

**Comment by AdmiringWorm** [comment]
No worries, no one can be expected to **_see**_ everything

File: new.xml [hunk 2]
```diff
@@ -188,4 +188,363 @@
...
+    <version os="Windows">
+      <locations>
+        <path ev="userdocuments" path="my games\StreetHacker"/>
+      </locations>
+      <files>
+        <include path="Database">
+          <!-- Unsure if this exclusion is needed, including since the size may become considerably high -->
+          <exclude path="Database\Backups"/>
+        </include>
+      </files>
```
Comment (AdmiringWorm) [inline new.xml:487]: Didn't include a `<linkable />` because of the exclusion path.

**Pull request closed without merging**